swatch-subscription-sync/deploy/clowdapp.yaml

```diff
@@ -1241,6 +1241,7 @@ objects:
         MCT2537F5
         MCT2537RN
         MCT2660
+        MCT2741
```

Comment:
should we add all other variants ? or do we have logic to handle it already ?

Reply:
We don't, do you have a list of the variants handy?

Reply:
nevermind, found it